x-pack/test/detection_engine_api_integration/security_and_spaces/group1/get_rules_info.ts

```diff
@@ -0,0 +1,122 @@
+
+import expect from '@kbn/expect';
+
+import { DETECTION_ENGINE_RULES_URL } from '@kbn/security-solution-plugin/common/constants';
+import { PREBUILT_RULES_URL } from '@kbn/security-solution-plugin/common/detection_engine/prebuilt_rules';
+import { RULES_INFO_URL } from '@kbn/security-solution-plugin/common/detection_engine/rule_management/api/urls';
+
+import { FtrProviderContext } from '../../common/ftr_provider_context';
+import {
+  createSignalsIndex,
+  deleteAllAlerts,
+  deleteSignalsIndex,
+  getSimpleRule,
+  deleteAllTimelines,
+} from '../../utils';
+
+// eslint-disable-next-line import/no-default-export
+export default ({ getService }: FtrProviderContext): void => {
+  const supertest = getService('supertest');
+  const es = getService('es');
+  const log = getService('log');
+
+  describe('get_rules_info', () => {
+    beforeEach(async () => {
+      await createSignalsIndex(supertest, log);
+    });
+
+    afterEach(async () => {
+      await deleteSignalsIndex(supertest, log);
+      await deleteAllAlerts(supertest, log);
+      await deleteAllTimelines(es);
+    });
+
+    it('should return expected JSON keys', async () => {
+      const { body } = await supertest
+        .get(RULES_INFO_URL)
+        .set('kbn-xsrf', 'true')
+        .send()
+        .expect(200);
+
+      expect(Object.keys(body)).to.eql([
+        'rules_custom_count',
+        'rules_prebuilt_installed_count',
+        'tags',
+      ]);
+    });
+
+    it('should return the correct result when there are no rules', async () => {
+      const { body } = await supertest
+        .get(RULES_INFO_URL)
+        .set('kbn-xsrf', 'true')
+        .send()
+        .expect(200);
+      expect(body.rules_custom_count).to.eql(0);
+      expect(body.rules_prebuilt_installed_count).to.eql(0);
+      expect(body.tags).to.eql([]);
+    });
+
+    describe('when there is a custom rule', () => {
+      beforeEach(async () => {
+        const rule = getSimpleRule();
+        rule.tags = ['tag-a'];
+
+        await supertest
+          .post(DETECTION_ENGINE_RULES_URL)
+          .set('kbn-xsrf', 'true')
+          .send(rule)
+          .expect(200);
+      });
+
+      it('should return the correct number of custom rules', async () => {
+        const { body } = await supertest
+          .get(RULES_INFO_URL)
+          .set('kbn-xsrf', 'true')
+          .send()
+          .expect(200);
+        expect(body.rules_custom_count).to.eql(1);
+        expect(body.rules_prebuilt_installed_count).to.eql(0);
+      });
+
+      it('should return correct tags', async () => {
+        const { body } = await supertest
+          .get(RULES_INFO_URL)
+          .set('kbn-xsrf', 'true')
+          .send()
+          .expect(200);
+        expect(body.tags).to.eql(['tag-a']);
+      });
+    });
+
+    describe('when there are installed prebuilt rules', () => {
+      beforeEach(async () => {
+        await supertest.put(PREBUILT_RULES_URL).set('kbn-xsrf', 'true').send().expect(200);
```

Comment:
Do we need to install _all_ the prebuilt rules for these cases? Would it be possible to test against a small controlled subset of prebuilt rules? Moreover, after [this PR](https://github.com/elastic/kibana/pull/143839) is merged, all prebuilt rules will no longer be available in tests.

Reply:
I've updated the tests according to [the merged back PR](https://github.com/elastic/kibana/pull/143839) which removed filesystem prebuilt rules.